src/groups/mqb/mqbnet/mqbnet_tcpsessionfactory.h

```diff
@@ -260,136 +260,109 @@ class TCPSessionFactory {
-    // 'd_noSessionCondition'
-    // condition variable), but it is
-    // declared atomic so that we can
-    // access it read-only outside
-    // the mutex for logging
-    // purposes.
 
+    /// Condition variable signaled after the last session created by this
+    /// factory has been destroyed.
     bslmt::Condition d_noSessionCondition;
-    // Condition variable signaled
-    // after the last session created
-    // by this factory has been
-    // destroyed.
 
+    /// Condition variable signaled after all clients and proxies are
+    /// destroyed.
     bslmt::Condition d_noClientCondition;
-    // Condition variable signaled
-    // after all clients and proxies
-    // are destroyed.
 
+    /// Map of all active channels
     ChannelMap d_channels;
-    // Map of all active channels
 
+    /// Manager of all open ports
     PortManager d_ports;
-    // Manager of all open ports
 
+    /// True if the recurring heartbeat check event is active.
     bool d_heartbeatSchedulerActive;
-    // True if the recurring
-    // heartbeat check event is
-    // active.
 
+    /// Scheduler handle for the recurring event used to heartbeat monitor the
+    /// channels.
     bdlmt::EventSchedulerRecurringEventHandle d_heartbeatSchedulerHandle;
-    // Scheduler handle for the
-    // recurring event used to
-    // heartbeat monitor the
-    // channels.
 
+    /// Map of all channels which are heartbeat enabled; only manipulated from
+    /// the event scheduler thread.
     bsl::unordered_map<bmqio::Channel*, ChannelInfo*> d_heartbeatChannels;
-    // Map of all channels which are
-    // heartbeat enabled; only
-    // manipulated from the event
-    // scheduler thread.
 
+    /// Value for initializing 'ChannelInfo.d_missedHeartbeatCounter'.  See
+    /// comments in 'calculateInitialMissedHbCounter'.
     const int d_initialMissedHeartbeatCounter;
-    // Value for initializing
-    // 'ChannelInfo.d_missedHeartbeatCounter'.
-    // See comments in
-    // 'calculateInitialMissedHbCounter'.
 
     /// Handles that can be used to stop listening. Empty unless listening.
     ListeningHandleMap d_listeningHandles;
 
+    /// Set to 'true' before calling 'listen'.  Set to 'false' in
+    /// 'stopListening'.
     bsls::AtomicBool d_isListening;
-    // Set to 'true' before calling
-    // 'listen'.  Set to 'false' in
-    // 'stopListening'.
 
+    /// Mutex for thread safety of this component.
     mutable bslmt::Mutex d_mutex;
-    // Mutex for thread safety of
-    // this component.
 
-    // Maintain ownership of 'OperationContext' instead of passing it to
-    // 'ChannelFactory::listen' because it may delete the context (on
-    // stopListening) while operation (readCallback/ negotiation) is in
-    // progress.
+    /// Maintain ownership of 'OperationContext' instead of passing it to
+    /// 'ChannelFactory::listen' because it may delete the context (on
+    /// stopListening) while operation (readCallback/ negotiation) is in
```

Comment:
```suggestion
    /// Maintain ownership of `OperationContext` instead of passing it to
    /// `ChannelFactory::listen` because it may delete the context (on
    /// stopListening) while operation (readCallback/negotiation) is in
```